### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by brianlehr**
Updated items

# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## As part of this Pull Request I have

- [ ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [ ] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by ejhenry** [comment]
@brianlehr - do you need help working through the failed linting checks or is this something you can take care of?

**Comment by JefferyMitchell** [comment]
Thank you @ejhenry for all your help!!

**Comment by ejhenry** [comment]
Can someone from @Azure/aprl-networking review this PR please?

**Review by ejhenry — APPROVED** [review]
LGTM 👍

**Comment by ejhenry** [comment]
@Azure/aprl-networking please review

**Comment by ehaslett** [comment]
> @Azure/aprl-networking please review

@brianlehr Pls see my pending question/review above

File: docs/content/services/networking/public-ip/_index.md [hunk 1]
```diff
@@ -63,7 +61,7 @@ Note - All basic SKU public IP addresses are created as non-zonal. Any IP that i
 
 **Guidance**
 
-Prevent risk of connectivity failures due to SNAT port exhaustion by using NAT gateway for outbound traffic from your virtual networks. NAT gateway scales dynamically and provides secure connections for traffic headed to the internet. We don't recommend exceeding 100 simultaneous outbound connections to a public IP address per worker. Avoid communicating with downstream services through public IP addresses when a private address (Private Endpoint) or Service Endpoint through vNet Integration could be used.
```
[the remark this answers is not in the document]
Comment (brianlehr) [reply docs/content/services/networking/public-ip/_index.md:66]: I am not aware of any "100 simultaneous connections" limit (also no idea where this "per worker" verbiage is coming from, I'm guessing this was copied and pasted in from somewhere).  I would delete that sentence.

File: docs/content/services/networking/public-ip/_index.md [hunk 2]
```diff
@@ -37,9 +37,7 @@ Definitions of states can be found [here]({{< ref "../../../_index.md#definition
...
-In regions without availability zones, all public IP addresses are created as non-zonal. Public IP addresses created in a region that is later upgraded to have availability zones remain non-zonal. A public IP's availability zone can't be changed after the public IP's creation.
-Note - All basic SKU public IP addresses are created as non-zonal. Any IP that is upgraded from a basic SKU to standard SKU remains non-zonal.
+A zone-redundant IP is created in all zones for a region and can survive any single zone failure. A zonal IP is tied to a specific availability zone, and shares fate with the health of the zone. A "non-zonal" public IP address is placed into a zone for you by Azure and doesn't give a guarantee of redundancy.  When utilizing a Public IP with resources that support zone resiliency (such as an Azure Load Balancer or Azure Firewall), it is recommended to use zone-redundant IPs in most cases.
 
 **Resources**
```
[the remark this answers is not in the document]
Comment (JefferyMitchell) [reply docs/content/services/networking/public-ip/_index.md:42]: I think I completed this on line 45 below. @brianlehr

**Review by JefferyMitchell — APPROVED** [review]

**Pull request merged**